src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -26,6 +26,7 @@
 
 namespace NuGet.PackageManagement.VisualStudio
 {
+
```

Comment:
If you don't need to make any other changes & run a new build, don't bother changing this, but please try to be careful to avoid these types of unnecessary whitespace changes.

Reply:
Updated